cves/CVE-2013-0848.yml

```diff
@@ -64,7 +64,9 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: The decode_init function in libavcodec/huffuv.c in FFmpeg efore 1.1 allows
```

Comment:
This is probably why the yaml is failing - use the pipe character for multiline strings